guides/theming-your-components.md

```diff
@@ -51,11 +51,19 @@ Define all styles unaffected by the theme in a separate file referenced directly
 
 #### 3. Include the theme mixin in your application
 Use the Sass `@include` keyword to include a component's theme mixin wherever you're already
-including Angular Material's built-in theme mixins. 
+including Angular Material's built-in theme mixins.  You will need to import directly to the file
+where your theme palette definitions have been made (typically styles.scss).  If you are creating 
+many custom component themes, you may wish to place them all in a single file in order to `@include`
+a single secondary theme, just as the Angular Material Theme includes styles for many components.
+This can avoid the additional complexity of creating a cascading chain of `map-get()` and `mat-color`
+commands to import your theme down a tree of imports, or having to `@include` multiple custom
+component themes for each alternate color theme defined for your application.
```

Comment:
I'm not sure about that this new content covers topics that belong in this guide. 

> You will need to import directly to the file where your theme palette definitions have been made

I think that explaining the import mixin is getting into basic Sass, which isn't really necessary here. I do think that it's good to show this in the example, though. 

> If you are creating many custom component themes, you may wish to place them all in a single file

I don't want to start giving advice here about organizing the theme mixins, since what we're doing isn't necessarily a best practice. If doing things over again, I probably wouldn't create a single mixin that pulls in the styles for all the components, instead having people include the individual components be be more explicit about dependencies.

Reply:
I'd be glad to remove the second sentence.  

As far as basic Sass is concerned, perhaps a better path would be to place a link outward to the Sass documentation?  The example prior to my edit included all the Sass -except- the import, which can be very confusing (eg following the example exactly and not getting the described result).   I wanted to make sure the example was fully functional.

Reply:
Yeah, including the import in the example is definitely an improvement. And linking to the Sass docs in this guide would be a good idea. Maybe something at the top like

"This document assumes working knowledge of Sass. Check out sass-lang.com/guide to lean more."